appdev.Dockerfile

```diff
@@ -1,25 +1,22 @@
-FROM buildpack-deps:focal
+FROM ubuntu:focal
 
 ### base ###
+ENV DEBIAN_FRONTEND=noninteractive
 RUN yes | unminimize \
     && apt-get install -yq \
+        curl \
```

Comment:
The [tips article](https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34) said not to use `curl`, but I got some weird errors during the build when not using it. The image size is getting much, much smaller already though, so I think this is okay.